README.md

```diff
@@ -1,29 +1,32 @@
 # Perma Lock
 
-`Perma Lock` is a contract that permanently locks a specific token, allowing users to add tokens but preventing any withdrawals. Once tokens are locked in this contract, they can't be removed.
+`Perma Lock FT` is a contract that permanently locks a specific token known at compile time, allowing users to add tokens but preventing any withdrawals. Once tokens are locked in this contract, they can't be removed. This contract is great for fungible tokens because as it minimizes the minimum required Lovelace and forces the minimum required Lovelace to be constant. The transaction fees are nearly constant as it strongly depends on the number of UTxOs inside the transaction and not necessarily the computational units.
```

Comment:
```suggestion
`Perma Lock FT` is a contract that permanently locks a specific token known at compile time, allowing users to add tokens but preventing any withdrawals. Once tokens are locked in this contract, they can't be removed. This contract is great for fungible tokens because it minimizes the minimum required Lovelace and forces the minimum required Lovelace to be constant. The transaction fees are nearly constant as it strongly depends on the number of UTxOs inside the transaction and not necessarily the computational units.
```

Reply:
First review in https://github.com/DripDropz/perma-lock/pull/2/commits/9b8cdcd0c3f31ab7ce79258f3a1646c9c4558113